docs/stdlib/datetime.rst

```diff
@@ -169,32 +167,30 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         Hint: Please use ISO8601 format. Alternatively "to_datetime"
         function provides custom formatting options.
 
-    All ``datetime`` values are restricted to the range from year 1 to 9999.
-
-    See functions :eql:func:`datetime_get`, :eql:func:`to_datetime`,
-    and :eql:func:`to_str` for more ways of working with
-    :eql:type:`datetime`.
-
+    All :eql:type:`datetime` values are restricted to a range of year 1 to
+    9999.
 
+    For more information regarding interacting with this type, see
+    :eql:func:`datetime_get`, :eql:func:`to_datetime` and :eql:func:`to_str`.
 
 
 ----------
 
 
 .. eql:type:: cal::local_datetime
 
-    A type representing date and time without time zone.
+    Represents a date and time without a given timezone.
 
-    :eql:op:`Casting <cast>` is a simple way to obtain a
-    :eql:type:`cal::local_datetime` value in an expression:
+    :eql:op:`Casting <cast>` is a simple way to obtain a value through an
+    expression:
 
     .. code-block:: edgeql
 
         select <cal::local_datetime>'2018-05-07T15:01:22.306916';
         select <cal::local_datetime>'2018-05-07T15:01:22';
 
-    Note that when casting from strings, the string should be in ISO
-    8601 format without timezone:
+    When casting :eql:type:`cal::local_datetime` from strings, the string must
+    follow the ISO8601 format with a timezone included.
```

Comment:
```suggestion
    follow the ISO8601 format with the timezone excluded:
```

The intent of the following example is to show that an ISO 8601 string cast to `local_datetime` that _does_ include a timezone will throw. Changing the text to reflect that and ending with a colon to indicate the example demonstrates the idea conveyed.